plugin/PDFDocumentPage.qml

```diff
@@ -40,19 +39,60 @@ DocumentPage {
         document: pdfDocument;
 
         onClicked: base.open = !base.open;
+
+        ViewPlaceholder {
+            enabled: pdfDocument.failure || pdfDocument.locked;
+            y: (flickable ? flickable.originY : 0)
+                   + (base.height - height - (passwd.visible ? passwd.height : 0)) / 2
+            text: if (pdfDocument.failure) {
+                //% "Broken file"
+                return qsTrId("sailfish-office-me-broken-pdf");
+            } else {
+                //% "Password protected file"
+                return qsTrId("sailfish-office-me-locked-pdf");
+            }
```

Comment:
These if/else could be replace with:

pdfDocument.failure ? qsTrId("sailfish-office-me-broken-pdf") : qsTrId("sailfish-office-me-locked-pdf")

Also please avoid using ending punctuation in qml code, is not needed at all, just produces more noise.